interface/ceed-basis.c

```diff
@@ -621,6 +623,75 @@ int CeedBasisCreateH1(Ceed ceed, CeedElemTopology topo, CeedInt num_comp,
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Create a non tensor-product basis for H(div) discretizations
+
+  @param ceed        A Ceed object where the CeedBasis will be created
+  @param topo        Topology of element (`CEED_QUAD`, `CEED_PRISM`, etc.), dimension of which is used in some array sizes below
+  @param num_comp    Number of components (usually 1 for vectors in H(div) bases)
+  @param num_nodes   Total number of nodes (dofs per element)
+  @param num_qpts    Total number of quadrature points
+  @param interp      Row-major (dim*num_qpts * num_nodes) matrix expressing the values of
+                       nodal basis functions at quadrature points
+  @param div        Row-major (num_qpts * num_nodes) matrix expressing
+                       divergence of nodal basis functions at quadrature points
+  @param q_ref       Array of length num_qpts holding the locations of quadrature
+                       points on the reference element
+  @param q_weight    Array of length num_qpts holding the quadrature weights on the
+                       reference element
+  @param[out] basis  Address of the variable where the newly created
+                       CeedBasis will be stored.
+  @return An error code: 0 - success, otherwise - failure
+  @ref User
+**/
+int CeedBasisCreateHdiv(Ceed ceed, CeedElemTopology topo, CeedInt num_comp,
+                        CeedInt num_nodes, CeedInt num_qpts, const CeedScalar *interp,
+                        const CeedScalar *div, const CeedScalar *q_ref,
+                        const CeedScalar *q_weight, CeedBasis *basis) {
+  int ierr;
+  CeedInt Q = num_qpts, P = num_nodes, dim = 0;
+  ierr = CeedBasisGetTopologyDimension(topo, &dim); CeedChk(ierr);
+  if (!ceed->BasisCreateHdiv) {
+    Ceed delegate;
+    ierr = CeedGetObjectDelegate(ceed, &delegate, "Basis"); CeedChk(ierr);
+
+    if (!delegate)
+      // LCOV_EXCL_START
+      return CeedError(ceed, CEED_ERROR_UNSUPPORTED,
+                       "Backend does not support BasisCreateHdiv");
```

Comment:
```suggestion
                       "Backend does not implement BasisCreateHdiv");
```
This will get GPU tests passing (because the string "Backend does not implement" is used to identify SKIP tests).